src/uid2Sdk.ts

```diff
@@ -40,12 +48,7 @@ export class UID2 extends SdkBase {
     return UID2.cookieName;
   }
   private static get Uid2Details(): ProductDetails {
-    return {
-      name: 'UID2',
-      defaultBaseUrl: 'https://prod.uidapi.com',
-      localStorageKey: 'UID2-sdk-identity',
-      cookieName: UID2.cookieName,
-    };
+    return productDetails;
```

Comment:
Is there a good reason why we follow a slightly different pattern than EUID? 
EUID looks like:
```
export class EUID extends SdkBase {
  private static cookieName = '__euid';
  // Deprecated. Integrators should never access the cookie directly!
  static get COOKIE_NAME() {
    console.warn(
      'Detected access to EUID.COOKIE_NAME. This is deprecated and will be removed in the future. Integrators should not access the cookie directly.'
    );
    return EUID.cookieName;
  }
  static get EuidDetails(): ProductDetails {
    return {
      name: 'EUID',
      defaultBaseUrl: 'https://prod.euid.eu',
      localStorageKey: 'EUID-sdk-identity',
      cookieName: EUID.cookieName,
    };
  }
...
```

Otherwise I think it would be better if we kept them consistent.

Reply:
Good catch! After talking with Lionell, we changed the pattern in UID2 and I forgot to update in EUID.  I've made them the same now, thanks so much